server/controllers/flights.js

```diff
@@ -4,27 +4,21 @@ module.exports = {
   create(req, res) {
     return Flight
       .create({
-        date: req.body.date,
-        duration: req.body.duration,
-        from: req.body.from,
-        to: req.body.to,
+        flightNumber: req.body.flightNumber,
```

Comment:
Same here 😢 Please make sure fKs exist as well, seems I can create with no problem something like this:

```
{
    "id": 1,
    "flightNumber": null,
    "departureDateTime": null,
    "arrivalDateTime": null,
    "airlineIndex": null,
    "destinationIndex": null,
    "originIndex": null,
    "updatedAt": "2018-10-29T21:37:12.012Z",
    "createdAt": "2018-10-29T21:37:12.012Z"
}
```

Reply:
I haven't check the rest of the endpoints, I assume they all need validation 🤔

Reply:
will look at it today during discrete maths